dev/integration_tests/flutter_gallery/lib/gallery/about.dart

```diff
@@ -66,7 +66,7 @@ void showGalleryAboutDialog(BuildContext context) {
               ),
               _LinkTextSpan(
                 style: linkStyle,
-                url: 'https://goo.gl/iv1p4G',
+                url: 'https://github.com/flutter/gallery',
```

Comment:
note for other reviewers, In addition to this removing a shortened url the previous url links to a dead location.